NetKVM/wlh/ParaNdis6_Driver.cpp

```diff
@@ -353,6 +354,7 @@ static VOID ParaNdis6_Halt(NDIS_HANDLE miniportAdapterContext, NDIS_HALT_ACTION
 {
     PARANDIS_ADAPTER *pContext = (PARANDIS_ADAPTER *)miniportAdapterContext;
     DEBUG_ENTRY(0);
+    pContext->systemThread.Stop();
```

Comment:
What about PM path like S4?

Reply:
Added commit for sync between power/reset flows and system thread actions